standalone-metastore/metastore-common/pom.xml

```diff
@@ -457,7 +457,7 @@
               <goal>run</goal>
             </goals>
             <configuration>
-              <protocArtifact>${protobuf.group}:protoc:${protobuf-exc.version}</protocArtifact>
+              <protocArtifact>com.google.protobuf:protoc:3.21.4</protocArtifact>
```

Comment:
Should we use `${protobuf.version}` instead of hardcoding `3.21.4`?

Reply:
Makes sense changed in the latest commit